include/common_structs.h

```diff
@@ -61,7 +61,9 @@ typedef struct Npc {
     /* 0x020 */ struct NpcBlurData* blurData; /* related to movement somehow... */
     /* 0x024 */ char unk_24[4];
     /* 0x028 */ u32 currentAnim;
-    /* 0x02C */ char unk_2C[12];
+    /* 0x02C */ char unk_2C[4];
+    /* 0x030 */ f32 unk_30;
+    /* 0x034 */ char unk_34[4];
     /* 0x038 */ struct Vec3f pos;
```

Comment:
I have a hunch unk_2C and unk_34 are also floats, but I don't know for sure, so they're still char arrays for now.